### ISSUE TITLE
`ZoomContentControl` does not work on WinAppSDK

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno.toolkit.ui#1291]
## Current behavior

Crashes with `Expected template part not found: PART_ContentGrid"`

## How to reproduce it (as minimally and precisely as possible)

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] WebAssembly
- [ ] Android
- [ ] iOS
- [ ] macOS (AppKit)
- [ ] Mac Catalyst
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Linux Framebuffer
  - [ ] Tizen
- [ ] Windows

IDE:

- [ ] Visual Studio 2022
- [ ] Visual Studio 2019
- [ ] Visual Studio Code
- [ ] Visual Studio for Mac
- [ ] Rider Windows
- [ ] Rider macOS

Relevant plugins:

**Comment by Xiaoy312** [comment]
so somehow the TemplateBinding doesnt work on windows... in this specific case...?
https://github.com/unoplatform/uno.toolkit.ui/blob/beb4d12a44eb6ee1d0a45ec2b3b771a7f2098c6b/src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.xaml#L36

given this xaml
```xml
<Grid>
	<ContentPresenter ...>
		<ContentPresenter.RenderTransform>
			<TransformGroup>
				<ScaleTransform ... ScaleX="{TemplateBinding ZoomLevel}" ScaleY="{TemplateBinding ZoomLevel}" />
				^ changing to binding with relativesource tempaltedparent made it work
	<StackPanel>
		<TextBlock Foreground="Red">tb ZoomLevel:<Run Text="{TemplateBinding ZoomLevel}" /></TextBlock>
		<TextBlock Foreground="Red">btp ZoomLevel:<Run Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=ZoomLevel}" /></TextBlock>
	</StackPanel>
```
here is the result on winappsdk:
![image](https://github.com/user-attachments/assets/7010ea11-5c13-48a2-b27b-cb9719aee9f6)

[microsoft says that "The types used as the value for the source property and the target property must match."](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/templatebinding-markup-extension) while it wasnt the case for the textblock, it should be for ScaleTransform.ScaleX and ZoomContentControl.ZoomLevel, both are `double` without nullable...

**Comment by vatsashah45** [comment]
Closing this issue as it has been resolved by #1297
